auth/src/shared/enums/errors.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum ERROR_MESSAGES {
```

Comment:
According to the known issue about ENUMs, we talked about not using ENUMs, I don't think this is an issue in the backend so we can leave it there. But frontend might just not use them, not an issue but something we can talk about in another meeting.